extensions/amp-consent/amp-consent.md

```diff
@@ -193,6 +193,33 @@ The prompt UI is defined within the consent instance config. The `promptUI` attr
 
 AMP displays prompt UI on page load or by user interaction. The prompt UI is hidden based on the three user actions described below.
 
+#### Prompt UI for Stories
+
+The `amp-story` extension provides a [default prompt UI](https://user-images.githubusercontent.com/1492044/40135514-8ab56d10-5913-11e8-95a2-72ac01ff31e0.png), that requires using a `<amp-story-consent>` component as the prompt UI. This component content (`title`, `message`, and a list of `vendors`) has to be specified in the main consent configuration.
+
+*Example*: Displays a prompt user interface on an AMP Story
+
+```html
+<amp-consent layout="nodisplay" id="consent-element">
+  <script type="application/json">
+  {
+    "consents": {
+      "my-consent": {
```

Comment:
@gmajoulet So there will be two consent instances created? I don't know how you implement it, but `<amp-consent>` doesn't support multiple consents right now. 
Is `story-consent` a reserved name now?

Reply:
I'm sorry, the example was wrong. The ``story-consent`` wasn't supposed to be nested in the ``consents`` property.